src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -483,6 +490,20 @@ export class EditConcept extends Component {
     return true;
   }
 
+  removeCurrentAnswer = async (info) => {
+    const { backgroundRemoveConceptMap, unPopulateAnswer } = this.props;
+    const data = {
+      references: [info.answerUrl],
+    };
+    try {
+      await backgroundRemoveConceptMap(data);
+      this.removeUnsavedMappingRow(info.answerUrl);
+      unPopulateAnswer(info.answer);
+    } catch (error) {
+      notify.show('An error occurred, can\'t edit this answer', 'error', 3000);
```

Comment:
Which kind of error is this?

Reply:
This notifies of any error in case the mapping fails to be removed. I've updated the message.